### PR TITLE
Build script

### PR DESCRIPTION
**Pull request by afishman**
Thus PR breaks down the `Dockerfile` into build scripts. Much easier to get the pipeline running natively (without Docker) now. 

- Individual `.bash` install files for each dependency
- `install-all-dependancies.bash` script
- Thin `Dockerfile`
- `install.bash` that SCE users can use to setup the pipeline to run natively in SCE3. I tested this on `CSU-002` but possible (probable) there are more be bugs.

### CODE REVIEW

File: docker/install-bwa.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# bwa
+git clone https://github.com/lh3/bwa.git
```
Comment (ellisrichardj) [inline docker/install-bwa.sh:2]: Could you change this to install a specific release of bwa?  We discussed this during the validation and this is the only tool where we don't have any control over the version used.    The latest release is: https://github.com/lh3/bwa/releases/download/v0.7.17/bwa-0.7.17.tar.bz2

**Comment by ellisrichardj** [comment]
I think this is a good approach and would make things easier to install without docker.  Just one comment about version control of bwa

File: docker/install-bwa.sh (hunk 1 above)
Comment (afishman) [reply docker/install-bwa.sh:2]: good spot, updated it to version pin at v0.7.17

Comment (ellisrichardj) [reply docker/install-bwa.sh:2]: Great thanks - just another item ticked off the TODO list

**Review by ellisrichardj — APPROVED** [review]
All looking good

**Pull request merged**